.drone.yml

```diff
@@ -106,6 +63,89 @@ steps:
     password:
       from_secret: docker_password
 
+---
+kind: pipeline
+name: test
+
+depends_on:
+- publish to docker hub
+
+steps:
+- name: download testdata
+  image: alpine/git:1.0.7
+  volumes:
+  - name: test-collections-v
+    path: /drone/src/testsuite/volumes/collections/testdata
+  commands:
+  - mkdir -p testsuite/volumes/collections
+  - git clone https://github.com/hoover/testdata testsuite/volumes/collections/testdata
+  - chown -R 666:666 testsuite/volumes/collections/testdata
+
+- name: run tests
+  image: liquidinvestigations/hoover-snoop2:${DRONE_COMMIT_BRANCH}
+  volumes:
+  - name: test-collections-v
+    path: /opt/hoover/collections/testdata
+# - name: dockersock
+#   path: /var/run
+# - name: dockersock
+#   path: /var/run/docker.sock
+# - name: gnupg-v
+#   path: /opt/hoover/gnupg
+# - name: snoop-blobs-v
+#   path: /opt/hoover/snoop/blobs
+# - name: search-v-testing
+#   path: /opt/hoover/search/hoover/site/settings/testing_local.py
+  environment:
+    WAIT_HOSTS: search-es:9200, snoop-pg:5432
+    WAIT_HOSTS_TIMEOUT: 60
+    SNOOP_DB: "postgresql://snoop:snoop@snoop-pg:5432/snoop"
+    SNOOP_COLLECTIONS: '[{"name": "testdata", "process": true}]'
+    SNOOP_AMQP_URL: "amqp://snoop-rabbitmq"
+    SNOOP_TIKA_URL: "http://snoop-tika:9998"
+    SNOOP_ES_URL: "http://search-es:9200"
+    SNOOP_COLLECTION_ROOT: "/opt/hoover/collections"
+    SNOOP_BLOB_STORAGE: "/opt/hoover/snoop/blobs"
+  commands:
+  - cd /opt/hoover/snoop
+  - py.test -vv
+  trigger:
+    events:
+    - push
+
+services:
+- name: snoop-rabbitmq
+  image: rabbitmq:3.7.3
+
+- name: snoop-tika
+  image: logicalspark/docker-tikaserver
+
+- name: search-es
+  image: docker.elastic.co/elasticsearch/elasticsearch:6.2.4
+  # volumes:
+  # - name: search-es-v
+  #   path: /drone/src/volumes/search-es/data
```

Comment:
I am not sure which path I have to specify. All the examples for postgres and elasticsearch I can find never have volumes.

Reply:
That's because they fill up the root volume that comes with the container. Leave it like this for now, will revisit in the future if it becomes a problem...